cmd/krel/cmd/release_notes.go

```diff
@@ -815,15 +810,25 @@ func releaseNotesJSON(tag string) (string, error) {
 		logrus.Infof("Release branch %s does not exist, working on %s", releaseBranch, git.DefaultBranch)
 	}
 
-	// If it's a patch release, we deduce the startTag manually:
+	// Notes for patch releases are generated starting from the previous patch release:
 	var startTag string
 	if tagVersion.Patch > 0 {
-		logrus.Debugf("Working on branch %s instead of %s", tag, git.DefaultBranch)
 		startTag = fmt.Sprintf("v%d.%d.%d", tagVersion.Major, tagVersion.Minor, tagVersion.Patch-1)
+		logrus.Infof("Using previous path release %s as start tag", startTag)
```

Comment:
```suggestion
		logrus.Infof("Using previous patch release %s as start tag", startTag)
```
I suggest combining those 3 identical log messages (down below) with a new one after the whole block.

Reply:
Done, there was a similar Info() already so I'm reusing that one.